packages/core/versioning/core-version-http-server/src/example.ts

```diff
@@ -21,26 +21,62 @@ const versionedRouter = vtk.createVersionedRouter({ router });
 // @ts-ignore unused variable
 const versionedRoute = versionedRouter
   .post({
-    path: '/api/my-app/foo/{name?}',
+    path: '/api/my-app/foo/{fooId?}/{name?}', // <= design mistake, {name?} should be a query parameter, but it has been released...
```

Comment:
How would this work if `fooId` isn't defined? In other words, would `/api/my-app/foo/undefined/name` or `/api/my-app/foo//name` work? Even if so, these look strange.